spec/models/assessment_spec.rb

```diff
@@ -4,6 +4,8 @@
 RSpec.describe Assessment, type: :model do
   let(:payload) { AssessmentRequestFixture.json }
 
+  it { is_expected.to have_many(:explicit_remarks) }
```

Comment:
not sure I'm a fan of these tests - they don't really test anything concrete (other than assessment has_many(:explicit_remarks)) which feels a bit pointless